### DISCUSSION TITLE
Tab management strategy

### DISCUSSION THREAD

**Discussion by 2007lab** [sumatrapdfreader/sumatrapdf#2431] [Ideas]
My Congrats on an awesome program. It has helped improve the volume of files and data I can quickly view.
That often leads to its own issues, such as having over 50 tabs open across different themes I am working or reading upon.
It would be greatly helpful to have some tab management/organizational capability and retention of such themes, when I need to get back to one. 
On Mozilla, there are many tab management features, where one can assign tabs to a theme/topic and then revert to them on another day.
I wish there was something similar in Sumatrapdfreader.
Currently, I save the advanced settings and then try to recover an old file. This succeeds only if said file is not in Windows temp folder.
thanks for reading.

**Comment by GitHubRulesOK** [comment]
Temp files or web cache files really should not be relied on finding again as the norm should be delete them frequently to not slow down constant virus checking etc. combined with that web cache filenames are usually transient [1] [2] ...

The filenames are remembered from session to session when remember opened files is active, but tabs clearly cannot be opened if the file is not at the stored fixed location, nor freshly downloaded into a tab from the web (that could pose a security risk). The best management strategy is rename the file to conventional folder like download ( or named folder within "MyDocuments" for a topic)

There are some ways a session can be controlled, for example SumatraPDF could use different -appdata setting files, so for one folders worth one folder name but different settings file for a different folder group, the downside is that does not allow for mixing two or more settings, unless you edit prior to launch.

Since windows includes many simple user based methods to manage files it can be easy to list a folder **or more** or mixed folder groups of pdfs and call that list to open all files on command or via shortcut. 

The requirements are as many permutations as users methods of squirrelling those file sets they require. But the solutions are also available many ways.
